.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ["3.9", "3.10", "3.11", "3.12", "3.13", "pypy-3.10"]
+        python: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13", "pypy-3.10"]
```

Comment:
why? 3.8 is very dead. really even supporting 3.9 is a bit silly but that's for another date.

Reply:
For some reason I thought 3.8 was still supported by CPython. I'll drop it.

(in defense of supporting many versions, it would be awful to have to upgrade a broken application in order to use a debug tool)